app/src/main/java/com/puzzlebench/clean_marvel_kotlin/presentation/extension/ImageExtensions.kt

```diff
@@ -1,8 +1,9 @@
 package com.puzzlebench.clean_marvel_kotlin.presentation.extension
 
 import android.widget.ImageView
+import com.bumptech.glide.Glide
 import com.squareup.picasso.Picasso
 
 fun ImageView.getImageByUrl(url: String) {
-    Picasso.with(context).load(url).fit().centerCrop().into(this)
+    Glide.with(context).load(url).centerCrop().into(this)
```

Comment:
change this to the common format convention.. 

Glide.with(context)
    .load(url)
    .centerCrop()
    .into(this)

Reply:
Done!